configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py

```diff
@@ -48,7 +48,7 @@
 ]
 test_pipeline = [
     dict(
-        type='SampleFrames',
+        type='DenseSampleFrames',
```

Comment:
Will this change affect testing results in README?

Reply:
No, it is a typo and doesn't affect the test results